src/Build/Microsoft.Build.csproj

```diff
@@ -1,4 +1,4 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
proably BOM changes, please get rid of them

Reply:
Done.